programs/marinade-finance/src/stake_system.rs

```diff
@@ -19,12 +20,17 @@ pub struct StakeRecord {
 impl StakeRecord {
     pub const DISCRIMINATOR: &'static [u8; 8] = b"staker__";
 
-    pub fn new(stake_account: &Pubkey, delegated_lamports: u64, clock: &Clock) -> Self {
+    pub fn new(
+        stake_account: &Pubkey,
+        delegated_lamports: u64,
+        clock: &Clock,
+        is_emergency_unstaking: u8,
```

Comment:
add the option to create a stake-account-record with the is_emergency_unstaking flag set